app/_includes/prereqs/install/amazon-linux.md

```diff
@@ -0,0 +1,27 @@
+{% navtabs %}
+{% navtab "Manual Installation" %}
+1. Download {{site.base_gateway}}
+```sh
+curl -Lo kong-enterprise-edition-3.9.0.1.rpm $(rpm --eval https://packages.konghq.com/public/gateway-39/rpm/amzn/%{amzn}/%{_arch}/kong-enterprise-edition-3.9.0.1.aws.%{_arch}.rpm)
+```
```

Comment:
Here and all other code blocks in an ordered list: make sure you indent the whole block so it doesn't break the ordered list.